packages/core/src/types.ts

```diff
@@ -198,20 +198,26 @@ export type SetupHandler<
     API extends EntityMap,
     EnvironmentContext extends Record<string, Context<any>>,
     Filter extends NormalizeEnvironmentFilter<EnvFilter> = NormalizeEnvironmentFilter<EnvFilter>
-> = (
-    feature: SettingUpFeature<ID, API, Filter>,
-    runningFeatures: RunningFeatures<Deps, Filter>,
-    context: MapRecordType<EnvironmentContext>
-) => RegisteringFeature<API, Filter>;
+    > = (
+        feature: SettingUpFeature<ID, API, Filter>,
+        runningFeatures: RunningFeatures<Deps, Filter>,
+        context: MapRecordType<EnvironmentContext>
+    ) => RegisteringFeature<API, Filter>;
 
 export type ContextHandler<
     C,
     EnvFilter extends EnvironmentFilter,
     Deps extends SomeFeature[],
     Filter extends NormalizeEnvironmentFilter<EnvFilter> = NormalizeEnvironmentFilter<EnvFilter>
-> = (runningFeatures: RunningFeatures<Deps, Filter>) => C;
+    > = (runningFeatures: RunningFeatures<Deps, Filter>) => C;
 
-export type PartialFeatureConfig<API> = Partial<MapToPartialType<JustFilter<API, Config<any, any>>>>;
+export interface Configurable<T> {
```

Comment:
is this enough to identify Configurable?

Reply:
seems to work

Reply:
I don't sure about this. maybe we should use a symbol or __kind .